### DISCUSSION TITLE
Support for individual playlists?

### DISCUSSION THREAD

**Discussion by shlomiv** [FreeTubeApp/FreeTube#4297] [Q&A]
Hi, brand new user here, just heard about this and installed. Looks amazing!

I have a bunch of playlists on my youtube, for different music styles, for different hobbies etc. I do see a "Playlists" tab, and when I save a video it goes there, but I don't see a way to create individual playlists.. Is that something that people are considering? 

I searched for this in the issues and discussions, but didn't find anything other than a single closed issue from 2019 that essentially said it could be easily possible, just not a priority. Wondered if this is a priority now.

Thanks!!

**Edit**
Just found the hoover tooltip
<img width="300" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/379063/914fbd10-29e6-4074-9290-67b42083c467">
This suggests it is on the roadmap, yay! Any rough projections when this would be available? 

Relevant [PR](https://github.com/FreeTubeApp/FreeTube/pull/4234) is waiting for merge. Maybe sooner then expected 🤞🤞

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No ETA but definitely next release but be patient though because that PR needs allot of testing

**Reply by shlomiv** [reply]
Sounds good, thank you! 🙏 😊
